app.js

```diff
@@ -8,31 +8,17 @@ Write a function called sum() that takes in two numbers as arguments and then re
 Test this function by hand in the console to get it working, and when you think it is finished, uncomment the call for the testSum() function below and check the console to see if the test passes.*/
 
 // Write your code here
- problem5
-// function sum(a, b) { //eslint-disable-line
-//   var theAnswer = a + b;
-//   return [theAnswer, 'The sum of ' + a + ' and ' + b + ' is ' + theAnswer + '.'];
-// }
 
-problem4
 function sum(a, b) { //eslint-disable-line
   var theAnswer = a + b;
   return [theAnswer, 'The sum of ' + a + ' and ' + b + ' is ' + theAnswer + '.'];
 }
- master
-// console.log("The sum of 4 and 7 is 11.")
-// console.log(sum(4,7)[1])
+
+console.log("The sum of 4 and 7 is 11.")
```

Comment:
It's not a big deal for this assignment, but in the future, it's nice to take out leftover `console.log`s from your code, and leave just the important functionality in place.